File: brainbox/length_last_word.py
def lengthOfLastWord(s: str) -> int:
    trimmed = s.rstrip()
    words = trimmed.split()
    last_word = words[-1]
    return len(last_word)

print(lengthOfLastWord("Hello World"))          # 5
print(lengthOfLastWord("luffy is still joyboy"))# 6

File: brainbox/length_last_word.js
function lengthOfLastWord(s) {
    const trimmed = s.trim();
    const words = trimmed.split(/\s+/);
    const lastWord = words[words.length - 1];
    return lastWord.length;
}

console.log(lengthOfLastWord("Hello World"));            // 5
console.log(lengthOfLastWord("luffy is still joyboy"));  // 6
